### ISSUE TITLE
T visible on Back to Top icon

### ISSUE THREAD

**Issue by webhick** [webhick/phpbb-Quatloosia#1]
The T overlap for accessibility is a light gray, which makes it slightly visible on the black "Back to Top" icon.
